### PR TITLE
ATD Handbook

### PR DESCRIPTION
**Pull request by w1l1ams**
Added foundational information regarding ATD in the handbook.

### CODE REVIEW

File: token/README.md [hunk 1]
```diff
@@ -3,6 +3,27 @@
 [[toc]]
 
 ## Algorithmic Token Distribution
+The MESG Foundation will release the circulating supply of MESG Tokens via a system called Algorithmic Token Distribution (ATD) to increase stability and transparency.
```
Comment (NicolasMahe) [inline token/README.md]:
```suggestion
The MESG Foundation will release the total supply of MESG Tokens via a system called Algorithmic Token Distribution (ATD) to increase stability and transparency.
```

File: token/README.md [hunk 2]
```diff
@@ -3,6 +3,27 @@
 [[toc]]
 
 ## Algorithmic Token Distribution
+The MESG Foundation will release the circulating supply of MESG Tokens via a system called Algorithmic Token Distribution (ATD) to increase stability and transparency.  
+
+#### Supply limitations
+
+- Software is used to determine the daily real circulating volume
```
Comment (NicolasMahe) [inline token/README.md]:
```suggestion
- The Daily Traded Volume (DTV) is calculated from the aggregation of the last 24h traded volume of the major crosses against the MESG Token.
```

File: token/README.md [hunk 3]
```diff
@@ -3,6 +3,27 @@
...
+- 0-6% of of the previous day's actual total traded volume (TDTV) will be released into circulating supply each day. The exact percentage fluctuates to maintian optimal market health and a minimum 20% supply for MESG Foundation reserves to align longterm incentives.
```
Comment (NicolasMahe) [inline token/README.md]:
```suggestion
- 0-6% of of the previous day's actual DTV will be released into circulating supply each day. The exact percentage fluctuates to maintain optimal market health.
```

File: token/README.md [hunk 4]
```diff
@@ -3,6 +3,27 @@
 [[toc]]
 
 ## Algorithmic Token Distribution
+The MESG Foundation will release the circulating supply of MESG Tokens via a system called Algorithmic Token Distribution (ATD) to increase stability and transparency.  
+
+#### Supply limitations
+
+- Software is used to determine the daily real circulating volume
+- 0-6% of of the previous day's actual total traded volume (TDTV) will be released into circulating supply each day. The exact percentage fluctuates to maintian optimal market health and a minimum 20% supply for MESG Foundation reserves to align longterm incentives. 
+- A smart contract is used to ensure that no more than 6% TDTV is released, including founder, advisor and whale investor payouts.
```
Comment (NicolasMahe) [inline token/README.md]:
```suggestion
- A smart contract is used to ensure that no more than 6% DTV is released, including the allocation for Sale, Team and Founders.
```

File: token/README.md [hunk 5]
```diff
@@ -3,6 +3,27 @@
...
+#### Disbursement details
+
+- Of the 0-6% TDTV released, 66.7% will be sold on exchanges at the current market price. The daily unsold tokens, in addition to the remaining 33.3%, will be made available to to the MESG Foundation for direct token distribution.
+- For recurring token payouts, the MESG/USD exchange rate will be calculated from the rate at 13:00 (UTC+7) on the first day of the given month from the aggregated exchange rate.
+- For non-recurring payouts made before 13:00 (UTC+7) on a given day, the rate will be calculated from the previous day’s aggregated exchange rate at 13:00 (UTC+7).
+- For non-recurring payouts made after 13:00 (UTC+7), the aggregated exchange rate at 13:00 (UTC+7) from the same day will be used.
+
+#### Transparency standards
+
+- The [ATD Dashboard]( https://atd.mesg.com/) is used to promote transparency within the MESG Community. It displays token quanitities within various wallets, in addition to rates of token dispersment.
```
Comment (NicolasMahe) [inline token/README.md]:
```suggestion
- The [ATD Dashboard](https://atd.mesg.com/) is used to promote transparency within the MESG Community. It displays token quantities within various wallets, in addition to rates of token dispersment. 
```

Comment (NicolasMahe) [reply token/README.md]: Is "dispersment" really a word?

File: token/README.md [hunk 6]
```diff
@@ -3,6 +3,27 @@
...
+
+- Of the 0-6% TDTV released, 66.7% will be sold on exchanges at the current market price. The daily unsold tokens, in addition to the remaining 33.3%, will be made available to to the MESG Foundation for direct token distribution.
+- For recurring token payouts, the MESG/USD exchange rate will be calculated from the rate at 13:00 (UTC+7) on the first day of the given month from the aggregated exchange rate.
+- For non-recurring payouts made before 13:00 (UTC+7) on a given day, the rate will be calculated from the previous day’s aggregated exchange rate at 13:00 (UTC+7).
+- For non-recurring payouts made after 13:00 (UTC+7), the aggregated exchange rate at 13:00 (UTC+7) from the same day will be used.
+
+#### Transparency standards
+
+- The [ATD Dashboard]( https://atd.mesg.com/) is used to promote transparency within the MESG Community. It displays token quanitities within various wallets, in addition to rates of token dispersment. 
+- Statistics on MESG's ATD Dashboard can be compared against public records such as Etherscan to boost trustworthiness and transparency with the community.
```
Comment (NicolasMahe) [inline token/README.md]:
```suggestion
- Statistics on MESG's ATD Dashboard can be compared against public records such as CoinMarketCap to boost trustworthiness and transparency with the community.
```

File: token/README.md (hunk 3 above)
Comment (NicolasMahe) [reply token/README.md]: I removed the part about the 20% of reserve. Should we add it back? If yes, I would like to also say that the 5% of partners and bounties are fully unlocked.

File: token/README.md (hunk 5 above)
Comment (w1l1ams) [reply token/README.md]: haha yes it is. I just spelled it wrong. It's "disbursement"

File: token/README.md (hunk 2 above)
Comment (w1l1ams) [reply token/README.md]: "Major crosses"? What does this mean?

File: token/README.md [hunk 7]
```diff
@@ -3,6 +3,27 @@
...
+#### Supply limitations
+
+- The Daily Traded Volume (DTV) is calculated from the aggregation of the last 24h traded volume of the major crosses against the MESG Token.
+- 0-6% of of the previous day's actual DTV will be released into circulating supply each day. The exact percentage fluctuates to maintain optimal market health.
+- A smart contract is used to ensure that no more than 6% DTV is released, including the allocation for Sale, Team and Founders.
+- Releasing MESG Tokens based on actual volume protects tokens in slow/bear markets by keeping released quantities low. More tokens will be released in faster markets. 
+
+#### Disbursement details
+
+- Of the 0-6% TDTV released, 66.7% will be sold on exchanges at the current market price. The daily unsold tokens, in addition to the remaining 33.3%, will be made available to to the MESG Foundation for direct token distribution.
```
Comment (antho1404) [inline token/README.md]:
```suggestion
- Of the 0-6% DTV released, 66.7% will be sold on exchanges at the current market price. The daily unsold tokens, in addition to the remaining 33.3%, will be made available to to the MESG Foundation for direct token distribution.
```

**Comment by antho1404** [comment]
@NicolasMahe a link to the atd dashboard and all information about the token and smartcontract etc should be great like that we could use this page to fill all the application form from the different exchanges

**Pull request merged**